lib/generators/onesie/task_generator.rb

```diff
@@ -21,15 +22,35 @@ def filename
         "#{Onesie::Manager.tasks_path}/#{task_version}_#{file_name}#{task_priority}.rb"
       end
 
+      def run_contents
+        read_template || DEFAULT_TEMPLATE
+      end
+
+      def read_template
+        task_template ? Onesie::TemplateReader.read_template(custom_template_path) : nil
+      end
+
+      def custom_template_path
+        Rails.root.join('onesie', 'templates', "#{task_template}.rb")
```

Comment:
Got the following error:
```
Errno::ENOENT: No such file or directory @ rb_sysopen - /app/onesie/templates/NewPermission.rb
```
which suggests we may need the following change:
```suggestion
        Rails.root.join('onesie', 'templates', "#{task_template.underscore}.rb")
```

Reply:
fwiw it's following a similar pattern that's used for [re]running a onesie (i.e. you don't specify DEV-1384-DoTheThing, you specify dev_1384_do_the_thing), but can update this.

Reply:
I see - I was using the example in the description above: 

``` bundle exec rake onesie:new['DEV-1234-add-new-foo-permission',,'NewPermission']```

I am fine with either.

Reply:
noting given my example was calling a onesie filename but this is referencing a template, I'll fix it such that you can specify the NewPermission way.